### PR TITLE
#126 Add tutorial

### PR DESCRIPTION
**Pull request by hilton**
Here’s a first basic tutorial. Before assigning this to wolf for translation, please could you let me know what you think about the new tutorial, and in particular:
- length
- use of full screenshots
- level of detail.

Note that I made a couple of changes in the _Tasks_ and _Cases_ chapters, to share screenshots. It occurs to me that if we have tutorials with big screenshots, it might improve the rest of the manual to have fewer screenshots there, and to link to relevant tutorials instead. I’m not sure, though.

### CODE REVIEW

**Comment by TimKam** [comment]
Great!
Imho:
- The length is fine.
- The use of full screenshots might be helpful for readers who are not yet familiar with the app.
- It's an easy and straight-forward read.

I agree that introducing the tutorials can help to keep the main manual lean. Of course, understanding the manual should not require reading the tutorials.

Details:
- _Perhaps_, a sentence about ad hoc vs process cases could help. I know, there is a link to the corresponding user manual section...
- > In the main menu, select Cases and then in the Cases of list, select Cases without a process from the bottom of the list.
  
  Perhaps it's better to have labels in bold print? "Cases of list" confused me for on the first glance. A small screenshot might help as well.

File: docs/tutorials/ad-hoc.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Using an ad hoc case for a document approval
+--------------------------------------------
+
+This tutorial introduces the simplest way to get started with Signavio Workflow.
+You can start without first defining a process by :ref:`start-adhoc`.
+
+Workflows are often used for document approvals, so this tutorial uses the example of approving a report, called *June report*.
+
+In the main menu, select *Cases* and then in the *Cases of* list, select *Cases without a process* from the bottom of the list.
+This opens :ref:`Cases <cases-overview>` view, which you can use to create and view :ref:`ad hoc cases <start-adhoc>`.
```
Comment (TimKam) [inline docs/tutorials/ad-hoc.rst]: **the** Cases view?

**Comment by hilton** [comment]
@TimKam I’ve added a commit to resolve the issues you spotted. For the reference to process cases, I don’t really want to have forward references or explanations that are unnecessary for the current tutorial, so I think this is a good example when a note is useful, to mention the other topic.

I don’t really want to switch to bold for labels, because I think it’s too jarring. In this case, the sentence needed rewriting to prevent the confusion.

I’m going to merge this, so the release notes for Friday evening’s release can link to the new docs.

**Comment by TimKam** [comment]
@wolfsoe Can you translate this on `master` ?

**Pull request merged**